features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -357,13 +357,15 @@
         <Capacity>100000</Capacity>
     </SessionContextCache-->
 
-    <Provisioning>
-        <EnableDefaultInboundUserProvisioningListener>true</EnableDefaultInboundUserProvisioningListener>
-    </Provisioning>
-
-
-    <!--<EventListenerOrderIDs>-->
-        <!--<EventListener type="UserOperationEventListener"-->
-                       <!--name="org.wso2.carbon.identity.workflow.mgt.impl.userstore.UserStoreActionListener" orderId="1"/>-->
-    <!--</EventListenerOrderIDs>-->
+    <EventListeners>
+        <EventListener type="UserOperationEventListener"
```

Comment:
'type' also should be the fully qualified interface name. Otherwise there can be conflicts if same interface names comes from two packages.